src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/HologramProjector.java

```diff
@@ -27,12 +31,18 @@
 
 public class HologramProjector extends SlimefunItem {
 
-    private static final String OFFSET_PARAMETER = "offset";
+    private static final String X_OFFSET_PARAMETER = "x_offset";
+    private static final String Y_OFFSET_PARAMETER = "y_offset";
+    private static final String Z_OFFSET_PARAMETER = "z_offset";
+    private static final int[] BORDER = {1, 3, 5, 7, 8, 10, 12, 14, 16, 17};
+
+    private static final ItemSetting<Double> maxDistance = new ItemSetting<>("max-horizontal-offset", 5.0);
```

Comment:
```suggestion
    private ItemSetting<Double> maxDistance = new ItemSetting<>("max-horizontal-offset", 5.0);
```
This should not be static let alone final.

Reply:
![image](https://user-images.githubusercontent.com/31554056/95660555-50431580-0aee-11eb-928d-7ad5a6dabe46.png)

Reply:
> ![image](https://user-images.githubusercontent.com/31554056/95660555-50431580-0aee-11eb-928d-7ad5a6dabe46.png)
> 

So what?

Reply:
This should definitely be final, just absolutely not static.

Reply:
![image](https://user-images.githubusercontent.com/31554056/95660697-2a6a4080-0aef-11eb-85b6-d64ee652a48f.png)

Reply:
> ![image](https://user-images.githubusercontent.com/31554056/95660697-2a6a4080-0aef-11eb-85b6-d64ee652a48f.png)
> 

And why do you even have that method static? You are overusing it...

Reply:
> > ![image](https://user-images.githubusercontent.com/31554056/95660555-50431580-0aee-11eb-928d-7ad5a6dabe46.png)
> 
> So what?

maxDistance is never modified